copy_with_style.js

```diff
@@ -20,44 +22,44 @@
 
 class Copy_With_Style {
 	element = null; // The element to copy to the clipboard (with style!)
-	button = null; // The button that, we attach a click even handler to to copy the the element to the clipboard
+	button = null; // The button that, we attach a click event handler to copy the element to clipboard
 	mode = null; // "attribute" (to inline all styles with a "style" attribute on each element) or "tag" (to include a "style" tag)
 	progress = null; // A progress element that is a sibling or child of the button by default but can be specified explicitly.
 	stylesheets = "inline";
 
 	// The <details> tag renders badly when inlined. Most mail readers show the summary AND details.
-	// So we might want to remplace the whole <details> tag with it's <summary> when inlining. Only
+	// So we might want to replace the whole <details> tag with it's <summary> when inlining. Only
 	// relevant when mode == "attribute"
 	summarize_details = true;
 
 	// Show a progress bar if we can find one as a sibling or child of element
 	show_progress = true;
 
 	// We wrap the nominated element in a div that is not in the DOM. We can copy the wrapper
-	// with the element or not. Matters little it's a simple div.'
+	// with the element or not. Matters little as it's a simple div.
 	copy_wrapper = true;
 
-	// Styling classes for the button. A supplie button is given these clases to communicate state of
+	// Styling classes for the button. These classes will be appended to the button representing state of
 	// the preparation and copy.
 	class_button = null;
 	class_preparing = null;
 	class_ready = null;
 
-	// Element exclusion overrides and augmentations. These are all function that are called
-	// if defined and passed an HTML element as their sole argument. Thy must return true or
+	// Element exclusion overrides and augmentations. These are functions that are called
+	// if defined and passed an HTML element as their sole argument. They must return true or
 	// false, true to exclude, false to keep.
 	//
-	// the extra_ pair are in addition tod efault implentations
+	// the extra_pair are in addition to default implementations
 	// the other two replace the default implementations
 	//
-	// Deep exclusions eclude the element and all its children.
-	// Shallow exclusion exlcude the element and graft its chidlren onto its parent.
+	// Deep exclusions exclude the element and all its children.
+	// Shallow exclusion exclude the element and graft its children onto its parent.
 	deep_exclusions = null;
 	shallow_exclusions = null;
 	extra_deep_exclusions = null;
 	extra_shallow_exclusions = null;
 
-	// If we are inlining styles on a huuuge element we may wish to permit a responsive UI while we're inlining it.
+	// If we are inlining styles on a huge element we may wish to permit a responsive UI while we're inlining it.
```

Comment:
He, he, I don't mind either way, but this was a pun as in a Trumpian Huuuuge ;-), but I have a page with a huuuge element.